feat: add deprecation and archival messages to commands and README (#142)
heroku/plugin-java: README.md

[![](https://badgen.net/github/license/heroku/plugin-java)](LICENSE)
[![CI](https://github.com/heroku/plugin-java/actions/workflows/ci.yml/badge.svg)](https://github.com/heroku/plugin-java/actions/workflows/ci.yml)

> **Note:** This repo is archived. For information about deploying Java applications with Heroku, refer to [Deploying JAR and WAR Files](https://devcenter.heroku.com/articles/deploying-jar-and-war-files).

Reviewer: ```suggestion
**Note:** We archived this repo. For more information about deploying Java apps with Heroku, see [Deploying JAR and WAR Files](https://devcenter.heroku.com/articles/deploying-jar-and-war-files).
```

Author: Done, thank you!
